geozero/Cargo.toml

```diff
@@ -11,7 +11,8 @@ license = "MIT/Apache-2.0"
 keywords = ["geo", "geojson", "gdal", "geos", "postgis"]
 
 [features]
-default = ["with-svg", "with-wkt", "with-geo", "with-geojson"]
+default = ["with-csv", "with-svg", "with-wkt", "with-geo", "with-geojson"]
```

Comment:
Inclusion as default feature disputable. The (also disputable) rule until now was: fhe feature either adds no dependency or the format is important enough for adding a light dependency.

Reply:
I'm happy to have it non-default in the geozero library for now. Note that I've enabled the feature in the geozero-cli. 

Does that sound alright?